bentoml/service.py

```diff
@@ -0,0 +1,18 @@
+from __future__ import annotations
+import bentoml
+from transformers import pipeline
+
+
+@bentoml.service(
+    resources={"cpu": "2"},
+    traffic={"timeout": 10},
+)
+class Summarization:
+    def __init__(self) -> None:
+        # Load model into pipeline
+        self.pipeline = pipeline('summarization')
+
+    @bentoml.api
+    def summarize(self, text: str) -> str:
+        result = self.pipeline(text)
+        return result[0]['summary_text']
```

Comment:
```suggestion
from __future__ import annotations
import bentoml

@bentoml.service(
    resources={"cpu": "2"},
    traffic={"timeout": 10},
)
class Summarization:
    def __init__(self) -> None:
        # Load model into pipeline
        pass

    @bentoml.api
    def summarize(self, text: str) -> str:
        return "Hello World"
```

This simplification removes the need for extra dependencies. I verified that this is still exploitable.